src/dmd/tokens.h

```diff
@@ -227,7 +227,7 @@ struct Token
     int isKeyword();
     const char *toChars() const;
 
-    static const char *toChars(TOK value);
+    static const char *toChars(unsigned char value);
```

Comment:
This is not a real regression with v2.089, but acute now for LDC. It's a `ubyte` on the D side and C++ `TOK` here is an `unsigned char` alias (but not for LDC, where I switched it to an enum on the C++ side too).